nextest-runner/src/runner.rs

```diff
@@ -670,24 +670,28 @@ impl<'a> TestRunnerInner<'a> {
                             false
                         };
 
-                        let _ = run_sender.send(InternalTestEvent::Slow {
-                            test_instance: test,
-                            retry_data,
-                            // Pass in the slow timeout period times timeout_hit, since stopwatch.elapsed() tends to be
-                            // slightly longer.
-                            elapsed: timeout_hit * slow_timeout.period,
-                            will_terminate,
-                        });
+                        if slow_timeout.grace_period != Duration::ZERO {
```

Comment:
```suggestion
                        if !slow_timeout.grace_period.is_zero()
```

Reply:
Ah! This is nice. I adjusted this in a few places.